test/new-e2e/tests/process/windows_test.go

```diff
@@ -147,5 +143,7 @@ func (s *windowsTestSuite) TestManualProcessCheckWithIO() {
 	check := s.Env().RemoteHost.
 		MustExecute("& \"C:\\Program Files\\Datadog\\Datadog Agent\\bin\\agent\\process-agent.exe\" check process --json")
 
-	assertManualProcessCheck(s.T(), check, true, "MsMpEng.exe")
+	// Check stats for Datadog agent process as it has IO stats more reliably populated than MsMpEng.exe
```

Comment:
The reason why we initially used a non-agent process instead of one of the agent processes was because we wanted to ensure that the process agent was able to collect IO stats from a process started by a non-agent user. In Linux, processes always have access to their own procfs entries, so it was a way to ensure the agent user permissions were set up correctly.

I'm not exactly sure how this translates to Windows, since the process agent runs as `NT AUTHORITY\SYSTEM`, as does `MsMpEng.exe`. I think that being able to collect IO stats from the agent process gives us reasonable confidence that all other processes would be able to be collected as well, and I want to document that explicitly in the PR for any curious minds from the future.

Reply:
Do you monitor other processes in other tests ! I agree with the considerations of this PR, you should have full control of the workload, aka the process that the agent is monitored.